refactor: `system/bootstrap.php` only loads files and registers autoloader (#5972)
codeigniter4/CodeIgniter4: user_guide_src/source/installation/upgrade_420.rst

Breaking Changes
****************

> - The ``system/bootstrap.php`` file no longer returns a ``CodeIgniter`` instance, and does not load ``.env`` file. The ``index.php`` and ``spark`` now do them. If you have code that expects these behaviors, it will no longer work and must be modified.

Reviewer: ```suggestion
- The ``system/bootstrap.php`` file no longer returns a ``CodeIgniter`` instance, and does not load the ``.env`` file (now handled in ``index.php`` and ``spark``). If you have code that expects these behaviors it will no longer work and must be modified.
```

> - The ``system/bootstrap.php`` file no longer returns a ``CodeIgniter`` instance, and does not load the ``.env`` file (now handled in ``index.php`` and ``spark``). If you have code that expects these behaviors it will no longer work and must be modified.

Reviewer 2: Please mention _why_ this change was made as well.

Author: Done.
